Src/Support/Google.Apis.Auth/SignedTokenVerification.cs

```diff
@@ -83,13 +88,13 @@ internal static class SignedTokenVerification
                 throw new InvalidJwtException("JWT must contain 'iat' and 'exp' claims");
             }
             // Check that the token was issued in the past.
-            var nowSeconds = (options.Clock.UtcNow - UnixEpoch).TotalSeconds;
-            if (nowSeconds + options.IssuedAtClockTolerance.TotalSeconds < signedToken.Payload.IssuedAtTimeSeconds.Value)
+            var utcNow = options.Clock.UtcNow;
+            if (utcNow + options.IssuedAtClockTolerance < signedToken.Payload.IssuedAt)
```

Comment:
Possibly change the checks a few lines above to check IssuedAt and ExpirationTime for null instead of the raw seconds values, just for clarity.
You could even have local non-nullable copies:

```csharp
DateTimeOffset issuedAt = signedToken.Payload.IssuedAt ?? throw new InvalidJwtException("...");
DateTimeOffset expiresAt = signedToken.Payload.ExpiresAt ?? throw new InvalidJwtException("...");
```

That way no-one needs to think about nullable logic :)